### ISSUE TITLE
update about design & info

### ISSUE THREAD

**Issue by rosclaritha** [phisoft/toursbloom-web#53]
- [ ] change design
- [ ] simplify the wording

![image](https://github.com/phisoft/toursbloom-web/assets/30463117/165bf24e-98a8-43b2-8bbc-25d2b301590d)

**Comment by abigailtaruc** [comment]
![image](https://github.com/phisoft/toursbloom-web/assets/110286983/20f71a7e-8393-40c6-9305-6ce5530f0a69)
![image](https://github.com/phisoft/toursbloom-web/assets/110286983/f0d548a7-737e-402e-b49d-128f96bdecbb)
